docker-compose.yml

```diff
@@ -32,6 +32,26 @@ services:
     # We have to wait until the database is up and running.
     # Database might already be running, so we need to do a cleanup first.
     command: /bin/sh -c "sleep 5 && psql -f cleanup.sql && psql -f schema.sql"
+  mysql:
```

Comment:
Would it be an idea to introduce [profiles](https://docs.docker.com/compose/compose-file/15-profiles/)?  
If you spin things up locally, it would be enough to have postgres running.  
For running tests you need mysql and sqlserver too.

Reply:
There are already some `depends_on` relations in place now, so if you [run the `irma` service](https://github.com/privacybydesign/irmago/blob/2e12f46a8315fc434c17a83d017478ac2e215c91/README.md#running) then `mysql` and `sqlserver` are not started. But indeed, if you'd run `docker-compose up` then all databases would spin up.

In most cases I only use `docker-compose up` if I want to run the tests using the debugger in my IDE, and then you actually want all databases to be up. We can introduce a profile for this and add that [here in the README](https://github.com/privacybydesign/irmago/blob/2e12f46a8315fc434c17a83d017478ac2e215c91/README.md#running-the-tests). I'm not sure whether the performance gain for developers outweighs the inconvenience of having to switch profiles.

Reply:
Okay, that makes sense. Keep it as it is 👍